src/index.tsx

```diff
@@ -256,10 +329,27 @@ export default class AppIntroSlider<ItemT = any> extends React.Component<
       return;
     }
     const lastIndex = this.state.activeIndex;
-    this.setState({activeIndex: newIndex});
+    this.setState({ activeIndex: newIndex });
     this.props.onSlideChange && this.props.onSlideChange(newIndex, lastIndex);
   };
 
+  _onMomentumScrollEndV = (e: { nativeEvent: NativeScrollEvent }) => {
```

Comment:
Since the only difference between this function and `_onMomentumScrollEndH` is the `y`/`x` parameter I would prefer to just have a single `_onMomentumScrollEnd` function which uses either `y` or `x` depending on the vertical prop

Reply:
okay i'll change that